mac

```diff
@@ -139,12 +139,11 @@ brew "imagemagick"
 # Testing
 brew "qt@5.5"
 
-# Programming languages and package managers
+# Programming language prereqs and package managers
 brew "libyaml" # should come after openssl
-brew "node"
-brew "rbenv"
-brew "ruby-build"
+brew "coreutils"
 brew "yarn"
+cask "gpgtools"
```

Comment:
Nice, was just reading @dbernheisel's comments about GPG for https://github.com/asdf-vm/asdf-nodejs as well.

Probably nice encouragement to have GPG installed on machines anyway. I've been meaning to sign my commits!

Possible to add a sentence or two about need for GPG to the commit message?